package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "assume-aws-role",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
Could you undo this change? When I merge your patch I'll update the version number and publish the new version on NPM